src/api/index.test.ts

```diff
@@ -322,6 +326,37 @@ describe('GET /getPositions', () => {
       data: TEST_POSITIONS_CELO,
     })
   })
+
+  for (const [userAgent, shouldIncludeAave] of [
```

Comment:
is the reason to use for loop over it.each is for better test names? or something else?

Reply:
Yes for better test name.